src/acceptance/README.md

```diff
@@ -50,7 +50,7 @@ The full set of config parameters is explained below:
 
 * `service_name` (required): The name of the registered auto-scaler service, use `cf marketplace` to determine the name.
 * `service_plan` (required): The plan name of the registered auto-scaler service, use `cf marketplace` to determine the plan.
-* `report_interval` (required): How frequently metrics are collected. This value must match the value configured in your deployment.
+* `aggregate_interval` (required): How frequently metrics are aggregated. This value must match the value configured in your deployment.
```

Comment:
It would be good to have aggregate_interval should be greater than 60 (seconds) or something that tells the user that it will be set to 60 if less than 60.